apps/web/src/common/modules/widgets/_helpers/widget-field-helper.ts

```diff
@@ -0,0 +1,11 @@
+export const sortWidgetTableFields = (fields: string[]) => {
+    const priorityFields = ['Date', 'Year', 'Month', 'Day'];
```

Comment:
![image](https://github.com/cloudforet-io/console/assets/50662170/9cc108cb-68af-4e05-ad2b-5bdcf82e3f17)
We have DATE_FIELD constant.